jolie/src/main/java/jolie/CommandLineParser.java

```diff
@@ -94,7 +94,7 @@
 	 * 
 	 * @return the arguments passed to the JOLIE program.
 	 */
```

Comment:
Please copy the javadoc comments to the corresponding methods in InterpreterParameters, or we will lose documentation. (I'd like to keep at least the one we have... :-) )